diff --git a/components/safe_browsing/content/browser/client_side_model_loader.cc b/components/safe_browsing/content/browser/client_side_model_loader.cc
index 7fbcb79da40246d9a77605dec15e85f041a45be4..dab2fb89e86250c955a0708b398cb9cc9ddc8202 100644
--- a/components/safe_browsing/content/browser/client_side_model_loader.cc
+++ b/components/safe_browsing/content/browser/client_side_model_loader.cc
@@ -164,6 +164,7 @@ void ModelLoader::StartFetch(bool only_from_cache) {
   // Start fetching the model either from the cache or possibly from the
   // network if the model isn't in the cache.
 
+  BRAVE_DISABLE_FETCH
   // TODO(nparker): If no profile needs this model, we shouldn't fetch it.
   // Then only re-fetch when a profile setting changes to need it.
   // This will save on the order of ~50KB/week/client of bandwidth.
